sql/core/src/test/scala/org/apache/spark/sql/execution/python/PythonForeachWriterSuite.scala

```diff
@@ -75,15 +78,20 @@ class PythonForeachWriterSuite extends SparkFunSuite with Eventually {
         tester = new BufferTester(memBytes, sleepPerRowReadMs)
         f(tester)
       } finally {
-        if (tester == null) tester.close()
+        if (tester != null) tester.close()
```

Comment:
e... So here's why all test failed with NPE.

Reply:
yes, but that NPE just covered up another NPE from `SparkEnv.get.serializerManager`